src/Microsoft.Extensions.Caching.Abstractions/CacheEntryExtensions.cs

```diff
@@ -134,6 +134,24 @@ public static class CacheEntryExtensions
         }
 
         /// <summary>
+        /// Sets the size of the cache entry.
```

Comment:
the cache entry Value.
The thing being cached, not the infrastructure around it.